docs/user_guide/i18n.rst

```diff
@@ -0,0 +1,61 @@
+Internationalization
+====================
+
+This theme contains translatable strings. Most strings will automatically be translated if the language is `supported <https://github.com/pydata/pydata-sphinx-theme/tree/master/pydata_sphinx_theme/locale>`__. To add another language, see :ref:`translating-the-theme`.
+
+The theme's :doc:`options<configuring>` also contain translatable strings, which require a different process. The following instructions assume that you store your translations in a ``locale`` directory under your documentation directory, and that you want to use ``theme`` as the name the message catalog for these strings.
```

Comment:
We should link to a useful introduction for Sphinx translation, in my experience most users don’t know how this works so I think this would helo

Reply:
I don't know that any are especially helpful. See e.g. https://www.sphinx-doc.org/en/master/usage/advanced/intl.html